scripts/setupvars/setupvars.sh

```diff
@@ -14,6 +14,7 @@ INSTALLDIR="${SCRIPT_DIR}"
 export INTEL_OPENVINO_DIR="$INSTALLDIR"
 
 # parse command line options
+unset python_version
```

Comment:
maybe we need to unset all script temporary vars at the end of the script?

Reply:
The python_version is special, since it is the only one that is modified by a commandline argument (namely `-pyver` in this case).  For other internal variables, it is possible to simply assign to the internal variable - and the script seems to do this properly.

Reply:
Please, fix the issue https://github.com/openvinotoolkit/openvino/actions/runs/6072122976/job/16471498118?pr=18592